stix_shifter_modules/guardium/configuration/lang_en.json

```diff
@@ -11,7 +11,7 @@
         },
         "help": {
             "label": "Need additional help?",
-            "description": "More details on the data source setting can be found in the specified link"
+            "description": "More details on the datasource setting can be found in the specified link"
```

Comment:
please revert back to `data source`. this is an cp4s requirement.

Reply:
done